.github/workflows/front-docs.yml

```
@@ -0,0 +1,120 @@
+  pull_request:
+    branches:
+      - main
+jobs:
+  generate-openapi:
+    runs-on: ubuntu-22.04
+    outputs:
+      source_branch: ${{ steps.branch.outputs.source_branch }}
+    steps:
+      - name: Determine source branch
+        id: branch
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)